src/vl-modal.src.js

```diff
@@ -18,14 +18,15 @@ import "/dist/modal.js";
  * @property {boolean} closable - Attribuut wordt gebruikt om aan te duiden dat de modal sluitbaar is.
  * @property {boolean} not-cancellable - Attribuut wordt gebruikt om aan te duiden dat de modal niet annuleerbaar is.
  * @property {boolean} not-auto-closable - Attribuut wordt gebruikt om aan te duiden dat de modal niet sluit bij het uitvoeren van een actie in de button slot.
+ * @property {boolean} allow-overflow - Attribuut wordt gebruikt om aan te duiden de inhoud van de modal uit de modal mag treden.
```

Comment:
Is het eigenlijk niet mogelijk om de datepicker aan te passen ipv. tegen de modal te moeten zeggen dat hij wel of geen overflow moet toelaten? Dus dat de dropdown eigenlijk een overlay wordt ipv. dat hij ruimte inneemt in de container (ik veronderstel nu even dat dit het probleem is, correct me if I'm wrong). Dat lijkt me namelijk het echte probleem.

Reply:
@SanderKleykens meer details in #67 het moet in ieder geval op modal niveau opgelost worden, de datepicker kan namelijk niet bepalen dat er buiten de modal container gerenderd kan worden.

Reply:
Maar, even los van de modal, als je een datepicker in een div steekt met "overflow: scroll", dan gaat hij bij het openen van de datepicker een scrollbar laten zien in de div en je zal geen kalender te zien krijgen (net getest).
Is dat dan geen bug? Want de native html5 datepicker blijft gewoon werken in die situatie.
Het lijkt me toch dat hij zijn kalender moet overlayen, en niet in dezelfde div inserten en die div laten groeien?

Reply:
De native datepicker wordt volgens mij door de browser gerenderd? Hierdoor zijn er geen problemen met element boxing. Alternatieve oplossing zou kunnen zijn dat de datepicker op root niveau fixed gerenderd wordt met een zeer grote z-index, maar dit gaat problemen geven voor het positioneren tijdens het scrollen.

Je kan het ook omgekeerd bekijken, als je op je element zet dat er geen overflow toegelaten is, of de overflow scroll moet zijn, wil je dan dat er een child element beslist om toch uit de container te laten renderen?

Eigenlijk hoort een datepicker niet echt thuis in een modal.

Reply:
> De native datepicker wordt volgens mij door de browser gerenderd? Hierdoor zijn er geen problemen met element boxing. Alternatieve oplossing zou kunnen zijn dat de datepicker op root niveau fixed gerenderd wordt met een zeer grote z-index, maar dit gaat problemen geven voor het positioneren tijdens het scrollen.

Ja... Nu, op root niveau zitten werken is ook maar vuil en speelt niet goed met de shadow dom. Dus een geweldige optie is dat nu ook niet.

> Je kan het ook omgekeerd bekijken, als je op je element expliciet zet dat er geen overflow toegelaten is of de overflow scroll moet zijn (beide zijn geen standaard gedrag), wil je dan dat er een child element beslist om toch uit de container te laten renderen?

Ja, dat is natuurlijk discussieerbaar. Maar ik volg daar zelf de browser in: uw user wilt de datum aanpassen, dus het is duidelijk dat dit zijn aandacht moet trekken en boven alles mag/moet komen. Dus in mijn optiek heeft dat dan precedence boven wat je instelt mbt. overflow.

Hoeft nu niets mee gedaan te worden, maar dan heb ik tenminste mijn voorkeur en denkwijze gedeeld 😃

Reply:
@SanderKleykens uiteraard. We zullen nog afwachten wat de andere reviewers vinden, afhankelijk daarvan kunnen we al dan niet een ticket maken om dit later verder te bekijken.